QandA/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Meetings(models.Model):
+    meeting_id = models.AutoField(primary_key=True)
+    HOMER = 'H'
+    PLATO = 'P'
+    JARVIS = 'J'
+    FRIDAY = 'F'
+    VENUE_CHOICES = (
+        (HOMER,'homer'),
+        (PLATO,'plato'),
+        (JARVIS,'jarvis'),
+        (FRIDAY,'friday'),
+    )
+    venue = models.CharField(max_length=50,choices=VENUE_CHOICES,default=JARVIS)
+    moderator_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    start_time = models.DateTimeField(help_text="Please use the following format: <em>YYYY-MM-DD</em>.")
+    end_time = models.DateTimeField(help_text="Please use the following format: <em>YYYY-MM-DD</em>.")
+
+
+class Question(models.Model):
+    question_id = models.AutoField(primary_key=True)
```

Comment:
This field shouldn't be here based on my previous review.
Ref :- https://docs.djangoproject.com/en/2.0/topics/db/models/#automatic-primary-key-fields

Reply:
Done!